assets/cases/scripting/bytecode/bytecode.scene

```diff
@@ -1,18 +1,19 @@
 [
   {
     "__type__": "cc.SceneAsset",
-    "_name": "",
+    "_name": "bytecode",
```

Comment:
Why this file changed so much? Did you use v3.8.0 editor?

Reply:
Yep, I used cocos creator 3.8.0 to open the